### ISSUE TITLE
'yes' is always called as though arg[0] is the util itself

### ISSUE THREAD

**Issue by Akeboshiwind** [samuela/rustybox#19]
This isn't the case if you call it through the raw binary, e.g:
```bash
$ rustybox yes test
```

This will print something like the following:
```bash
yes test
yes test
yes test
```

Where it should be printing the following instead:
```bash
test
test
test
```

This is because it doesn't use the args that were passed into the function but instead get's the args from the `env` crate directly.

**Comment by samuela** [comment]
Yeah, this actually came up in https://github.com/samuela/rustybox/pull/7. I have a plan to refactor things to make this work, but I don't have access to my linux build machine at the moment. I'll get to it as soon as I can though!

**Comment by samuela** [comment]
@Akeboshiwind Ok, thanks for your patience! Should be fixed in 839c401da561d8085af5dd6149dd13202c9beb62 and some tests added in 2f62cd2fbec4d040a4fde07222fdffd91c980677.
